docs/source/index.rst

```diff
@@ -129,7 +129,7 @@ Documentation
    reference/interactive-nodes
    reference/faq
    reference/logging
-   reference/local/index
```

Comment:
Do we want to directly hide, or add a deprecated warning first?

Reply:
I'm in favor of directly hiding since it still runs the risk of people confusing it as the way to run on-prem and trying it out. People who still need access to the old onprem documentation can refer to docs for older versions on readthedocs.